pxf/pxf-service/src/scripts/pxf-service

```diff
@@ -216,17 +174,16 @@ function doInit()
 	determineHadoopDistro
 	generatePrivateClasspath || return 1
 	createInstance || return 1
-	configureInstance || return 1
 	deployWebapp || return 1
 	createLogsDir || return 1
 	createRunDir  || return 1
 }
 
 #
-# patchWebapp patches the webapp config files
-# patch applied only if PXF_HOME is defined
+# configureWebapp patches the webapp with pxf and user overriden configs
+# applied only if PXF_HOME is defined
 #
-function patchWebapp()
+function configureWebapp()
```

Comment:
since this function is now called on every start, we should not be unpacking WAR file and edit web.xml -- only catalina files that use dynamic properties should be regenerated. Ideally, they should be regenerated from templates that do not get destroyed, so that customers have a chance to change the template, if needed.

Reply:
The purpose of patching the war is so that an update done to PXF as part of upgrades wouldn't require the user to run pxf init again. They can simply use pxf restart.
Can evaluate that change outside of this PR